### PR TITLE
Director maps

### PR DESCRIPTION
**Pull request by CannonLock**
Many small patches and one new feature
- Restructure Config page so UI load happens in one step
- Move to one ButtonLink element for sidebar additions
- Add pointer to main dashboard to all servers
- Add Maps to director, both server level and overview
  - Overview map is very introductoryz
- Change the view of the Origin capabilities to better reflect the state

### CODE REVIEW

**Comment by CannonLock** [comment]
Screenshot 2024-07-17 at 2 51 05 PM (2)
Screenshot 2024-07-17 at 2 35 40 PM (2)

**Comment by haoming29** [comment]
@CannonLock Do we have an issue for this PR? Trying to figure out the objectives

**Review by haoming29 — CHANGES_REQUESTED** [review]
Some initial issues spotted:
* The maps for individual servers can't be zoomed in, which would be nice to figure out the location of the server. 
* The general map doesn't close the popover window if I click the empty area on the map.
* There's extra trailing `TEst` to the names of the server 
Screenshot 2024-07-30 at 11 08 43 AM

File: web_ui/frontend/app/director/layout.tsx [hunk 1]
```diff
@@ -39,15 +40,15 @@ export default function RootLayout({
...
-        <Box pt={1}>
-          <Tooltip title={'Config'} placement={'right'}>
-            <Link href={'/config/'}>
-              <IconButton>
-                <BuildIcon />
-              </IconButton>
-            </Link>
-          </Tooltip>
-        </Box>
+        <ButtonLink title={'Dashboard'} href={'/director'}>
```
Comment (haoming29) [inline web_ui/frontend/app/director/layout.tsx:43]:
```suggestion
        <ButtonLink title={'Dashboard'} href={'/director/'}>
```

File: web_ui/frontend/components/layout/OriginSidebar.tsx [hunk 2]
```diff
@@ -35,37 +35,17 @@ export const OriginSidebar = () => {
 
   return (
     <Sidebar>
+      <ButtonLink title={'Dashboard'} href={'/origin'}>
```
Comment (haoming29) [inline web_ui/frontend/components/layout/OriginSidebar.tsx:38]:
```suggestion
      <ButtonLink title={'Dashboard'} href={'/origin/'}>
```

File: web_ui/frontend/components/layout/OriginSidebar.tsx [hunk 3]
```diff
@@ -35,37 +35,17 @@ export const OriginSidebar = () => {
...
-            <Tooltip title={'Globus Configurations'} placement={'right'}>
-              <Link href={'/origin/globus/'}>
-                <IconButton>
-                  <Public />
-                </IconButton>
-              </Link>
-            </Tooltip>
-          </Box>
-        </>
+        <ButtonLink title={'Globus Configurations'} href={'/origin/globus'}>
```
Comment (haoming29) [inline web_ui/frontend/components/layout/OriginSidebar.tsx:42]:
```suggestion
        <ButtonLink title={'Globus Configurations'} href={'/origin/globus/'}>
```

File: web_ui/frontend/components/layout/OriginSidebar.tsx [hunk 4]
```diff
@@ -35,37 +35,17 @@ export const OriginSidebar = () => {
...
-      <Box pt={1}>
-        <Tooltip title={'Config'} placement={'right'}>
-          <Link href={'/config/'}>
-            <IconButton>
-              <Build />
-            </IconButton>
-          </Link>
-        </Tooltip>
-      </Box>
+      <ButtonLink title={'Config'} href={'/config'}>
```
Comment (haoming29) [inline web_ui/frontend/components/layout/OriginSidebar.tsx:46]:
```suggestion
      <ButtonLink title={'Config'} href={'/config/'}>
```

File: web_ui/frontend/components/Map/ServerMap.tsx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+      {showPopup && (
+        <Popup
+          longitude={server.longitude}
+          latitude={server.latitude}
+          closeOnClick={false}
+          onClose={() => setShowPopup(false)}
+          offset={[0, -24] as [number, number]}
+        >
+          <Box>
+            <Typography variant={'body1'}>{server.name} TEst</Typography>
```
Comment (haoming29) [inline web_ui/frontend/components/Map/ServerMap.tsx:57]:
```suggestion
            <Typography variant={'body1'}>{server.name}</Typography>
```

Comment (haoming29) [reply web_ui/frontend/components/Map/ServerMap.tsx:57]: Also, would it be easy to use the same server card for the list page here? We can do this in a separate PR if that's not a easy addition

Comment (CannonLock) [reply web_ui/frontend/components/Map/ServerMap.tsx:57]: Created new issue for this.

**Comment by CannonLock** [comment]
@haoming29 Fixed your three notes on the top as well. I can't add scroll to zoom on the map because it will catch when I am trying to scroll the list, but I added some buttons.

**Comment by haoming29** [comment]
@CannonLock can you fix the prettier linter error?

**Review by haoming29 — APPROVED** [review]
Changes LGTM! Good to merge if the prettier error is fixed

**Pull request merged**